ayon_bin_distro/gui/progress_ui.py

```diff
@@ -146,17 +179,28 @@ def _close_window(self):
 class ProgressDialog(QtWidgets.QDialog):
     def __init__(
         self,
-        controller,
+        controller: worker.Controller,
         delet_progress_bar_on_finish: bool = True,
         close_on_finish: bool = False,
         auto_close_timeout: float = 0.5,
         title: str = "",
+        ui_update_reate: int = 100,
```

Comment:
`reate`? :)

Did you mean `rate`? Would `interval` be more appropriate?

`ui_update_interval` or `ui_refresh_rate`?

Reply:
done in external commit.